streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -815,19 +815,25 @@ record = null;
         return true;
     }
 
-    private void handleException(final Throwable e) {
-        final StreamsException error = new StreamsException(
+    private void handleException(final Throwable originalException) {
+        handleException(
             String.format(
-                "Exception caught in process. taskId=%s, processor=%s, topic=%s, partition=%d, offset=%d, stacktrace=%s",
```

Comment:
given what we pass in `originalException` into `StreamsException, it seems not necessary to include the stracktrace in the error message.

Reply:
this seems to break two unit tests in ProcessingExceptionHandlerIntegrationTest while testing exception.getMessage

Reply:
Thanks! -- I did only run a few test locally, not the whole suite. Will take a look.